tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -130,7 +156,13 @@ protected:
    std::vector<ReadCallback_t> fReadCallbacks;
    /// C++ type version cached from the descriptor after a call to `ConnectPageSource()`
    std::uint32_t fOnDiskTypeVersion = kInvalidTypeVersion;
+   /// If set, the column representation used for serialization; otherwise the default from
+   /// GetColumnRepresentations() is used
+   std::unique_ptr<std::vector<EColumnType>> fColumnRepresentative;
```

Comment:
This is always an element in the list of column representations returned by `GetColumnRepresentations()`.  Any chance that we can avoid the copy and keep only a pointer/reference to it?